src/ApplicationInsights.ServiceFabric.Native/Net45/ServiceRemotingLoggingStrings.cs

```diff
@@ -5,5 +5,6 @@ internal static class ServiceRemotingLoggingStrings
         public const string ParentIdHeaderName = "Microsoft.ApplicationInsights.ServiceFabric.ParentId";
         public const string CorrelationContextHeaderName = "Microsoft.ApplicationInsights.ServiceFabric.CorrelationContext";
         public const string ServiceRemotingTypeName = "ServiceFabricServiceRemoting";
+        public const string ServiceRemotingTypeNameTracked = "ServiceFabricServiceRemoting (tracked component)";
```

Comment:
Does app map knows if it can handle this type? Can you check with Zhenhua/Yanming?